screens/AppLayout.tsx

```diff
@@ -31,6 +28,13 @@ export const AppLayout: React.FC = () => {
         ))}
         {controller.isAuthorized &&
           authRoutes.map((route) => <Screen key={route.name} {...route} />)}
+        <MessageOverlay
+          isVisible={controller.isReadError}
+          title="Failed to read keys"
```

Comment:
we should read the literals from the locales files, instead of hardcoding the literals.

Reply:
I'll use this key and put it in under `WelcomeScreen` JSON key in the `<lang>.json` just like how MessageOverlay is being used everywhere.